scripts/variantstore/tieout/FULL_TIEOUT.md

```diff
@@ -0,0 +1,92 @@
+# NOTES
+ - excess het sites are PRESENT but FILTERED (ExcessHet) in WARP pipeline
+ - excess het sites are NOT MARKED OR FILTERED IN BQ PIPELINE
+ - python strips out INFO field annotations with no `=` part (no effect)
+ - Previously there was an attempt to tie out the feature extract input to VQSR between WARP and GVS.  However, that was abandoned in favor of the truth-sample based comparison here due to non-determinism and bugs in GenomicsDB that were hard to compensate for.  However, the docs and scripts for doing this are in the branch `gvs_vqsr_input_tieout` under `scripts/variantstore/tieout/feature_extract`
+ 
+## One time tasks
+```
+REFERENCE="/Users/kcibul/projects/references/hg38/v0/Homo_sapiens_assembly38.fasta"
+
+gsutil -m cp gs://dsde-palantir/NISTTruthData/* .
+gatk IntervalListToBed -I NIST_highconfidenceregions.interval_list -O NIST_highconfidenceregions.bed
+rtg format --output human_REF_SDF $REFERENCE
+```
+## WARP -- take full results, extract two samples, and strip down to chr20
+```
+WORKFLOW_ID=36e7547e-3253-4888-9b1c-2a7437401aee
+gsutil -m cp gs://broad-dsp-spec-ops-cromwell-execution/JointGenotyping/${WORKFLOW_ID}/call-FinalGatherVcf/warp_tieout_acmg_cohort_v1.vcf.gz .
+
+~/gatk SelectVariants -V warp_tieout_acmg_cohort_v1.vcf.gz --sample-name SM-G947Y --select-type-to-exclude NO_VARIATION -O NA12878.warp.vcf.gz
+bcftools view -O z NA12878.warp.vcf.gz chr20 > NA12878.warp.chr20.vcf.gz
+tabix NA12878.warp.chr20.vcf.gz
+
+~/gatk SelectVariants -V warp_tieout_acmg_cohort_v1.vcf.gz --sample-name CHMI_CHMI3_WGS1 --select-type-to-exclude NO_VARIATION -O SYNDIP.warp.vcf.gz
+bcftools view -O z SYNDIP.warp.vcf.gz chr20 > SYNDIP.warp.chr20.vcf.gz
+tabix SYNDIP.warp.chr20.vcf.gz
+
+gsutil -m cp NA12878* gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/warp/
+gsutil -m cp SYNDIP* gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/warp/
+
+```
+
+## BIGQUERY 
+
+First, create a full cohort extract (as described in README.md) using the `gvs_tieout_acmg_v3` (baseline), or otherwise desired, filtering model.  
+
+```
+~/gatk SelectVariants -V gvs.bq.all.noeh.vcf.gz --sample-name SM-G947Y --select-type-to-exclude NO_VARIATION -O NA12878.bq.all.noeh.vcf.gz
+bcftools view -O z NA12878.bq.all.vcf.gz chr20 > NA12878.bq.all.chr20.vcf.gz
+tabix NA12878.bq.all.chr20.vcf.gz
+
+~/gatk SelectVariants -V gvs.bq.all.vcf.gz --sample-name CHMI_CHMI3_WGS1 --select-type-to-exclude NO_VARIATION -O SYNDIP.bq.all.vcf.gz
```

Comment:
did we only filter out excess het for NA12878 and not for CHMI?

Reply:
oh I just cleaned up the examples here.  I'm hopeful the next time we run this we will have EH built in